includes/amp-helper-functions.php

```diff
@@ -491,6 +491,48 @@ function amp_filter_script_loader_tag( $tag, $handle ) {
 	return $tag;
 }
 
+/**
+ * Opt-in to CORS Mode for the font stylesheets.
+ *
```

Comment:
[Just a suggestion] We may simplify this comment by directly specifying the action: 

"Explicitly opt-in to CORS mode by adding the crossorigin attribute to font stylesheet links. This explicitly triggers a CORS request, and gets back a non-opaque response, ensuring that a service worker caching the external stylesheet will not inflate the storage quota."